bluebrain/repo-bluebrain/packages/py-neurots/package.py

```diff
@@ -23,7 +23,7 @@ class PyNeurots(PythonPackage):
     depends_on("py-setuptools-scm", type="build")
 
     depends_on("py-diameter-synthesis@0.5.4:", type=("build", "run"), when="@3.4:")
-    depends_on("py-diameter-synthesis@0.5.3:", type=("build", "run"), when="@:3.3")
+    depends_on("py-diameter-synthesis@0.5.2:", type=("build", "run"), when="@3.3.1:")
```

Comment:
Isn't it an issue if the `when` ranges overlap @matz-e ?

Reply:
The requirement is updated/overwritten for `0.5.4`. The main point was that according to `setup.py`, `diameter-synthesis` is only required from 3.3 onwards.